.idea/ehrenamtskarte.iml

```diff
@@ -2,11 +2,7 @@
 <module type="WEB_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
-    <content url="file://$MODULE_DIR$">
-      <excludeFolder url="file://$MODULE_DIR$/administration" />
-      <excludeFolder url="file://$MODULE_DIR$/backend" />
-      <excludeFolder url="file://$MODULE_DIR$/frontend" />
-    </content>
+    <content url="file://$MODULE_DIR$" />
```

Comment:
Hmm... I think these changes should not be necessary....

Reply:
They also do not harm :) i tested this. No differnce with or without these changes.